### ISSUE TITLE
Regarding the type when setting an alias.

### ISSUE THREAD

**Issue by yuuri111** [francescov1/mongoose-tsgen#122]
Hello.  I am having issues trying to set an alias.
```
"node":"16.19"
"npm": "^9.5.0",
"mongoose": "^6.7.2",
"mongoose-tsgen": "^9.1.0"
```

When setting an alias, I believe the type becomes 'any'. Is there a way to set the type to the one specified by 'type'?

ex)
models/test.ts
```
interface TestSchemaFields extends Document {
  _id: string
  objectId: string
}

const TestSchema = new Schema<TestSchemaFields>({
  _id: {
    type: String,
    required: true,
    unique: true,
    alias: 'objectId',
  },
})
```

mongoose-types.ts
```
export type TestDocument = mongoose.Document<string, TestQueries> &
  TestMethods & {
    _id: string
    objectId: any
  }
```

**Comment by francescov1** [comment]
Hey @yuuri111, thanks for submitting. The library doesn't currently have any handling for aliased fields, which is why its being typed as any. I dont think it should be too hard to add, will keep you updated!

**Comment by francescov1** [comment]
Fix is live in v9.2.6 🚀
